app/src/main/kotlin/com/mapzen/erasermap/view/InstructionAdapter.kt

```diff
@@ -11,8 +11,8 @@ import com.mapzen.erasermap.util.DisplayHelper
 import com.mapzen.valhalla.Instruction
 import java.util.ArrayList
 
-public class InstructionAdapter(val context: Context, val instructions: ArrayList<Instruction>,
-        val pager: RouteModeView) : PagerAdapter() {
+public class InstructionAdapter(private val context: Context,
```

Comment:
Excuse the kotlin-101 question but what does making theses arguments private do? Are they public by default?